lightning/src/ln/channelmanager.rs

```diff
@@ -3797,6 +3803,16 @@ impl<'a, ChanSigner: ChannelKeys + Readable, M: Deref, T: Deref, K: Deref, F: De
 			per_peer_state.insert(peer_pubkey, Mutex::new(peer_state));
 		}
 
+		let max_alloc_size: u64 = 1024 * 64;
```

Comment:
You probably want to just make this a const (which rustc will complain if its not IN_ALL_CAPS.